airflow/jobs/scheduler_job.py

```diff
@@ -433,6 +433,22 @@ def is_alive(self, grace_multiplier=None):
             (timezone.utcnow() - self.latest_heartbeat).seconds < scheduler_health_check_threshold
         )
 
+    def succeed_recently(self):
+        """
+        Did this SchedulerJob succeed recently?
+
+        We define the job succeeded recently if the job state is success
+        and ends within the threshold defined in the ``scheduler_health_check_threshold``
+        config setting.
+
+        :rtype: boolean
+        """
+        scheduler_health_check_threshold = conf.getint('scheduler', 'scheduler_health_check_threshold')
+        return (
+            self.state == State.SUCCESS and self.end_date
+            (timezone.utcnow() - self.end_date).seconds < scheduler_health_check_threshold
```

Comment:
Logic looks good, but I'm confused by the syntax. Don't we need another `and` here?

Reply:
oh. my obvious typo.